Makefile

```diff
@@ -1,25 +1,51 @@
-# Current Operator version
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 2.0.0-snapshot
-# Default bundle image tag
-BUNDLE_IMG ?= quay.io/kiegroup/kogito-operator-bundle:$(VERSION)
-# Default catalog image tag
-CATALOG_IMG ?= quay.io/kiegroup/kogito-operator-catalog:$(VERSION)
-# Default bundle image tag
-PROFILING_IMG ?= quay.io/kiegroup/kogito-operator-profiling:$(VERSION)
-# Options for 'bundle-build'
+
+# CHANNELS define the bundle channels used in the bundle.
+# Add a new line here if you would like to change its default config. (E.g CHANNELS = "candidate,fast,stable")
+# To re-generate a bundle for other specific channels without changing the standard setup, you can:
+# - use the CHANNELS as arg of the bundle target (e.g make bundle CHANNELS=candidate,fast,stable)
+# - use environment variables to overwrite this value (e.g export CHANNELS="candidate,fast,stable")
 CHANNELS=alpha,1.x
+ifneq ($(origin CHANNELS), undefined)
 BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+
+# DEFAULT_CHANNEL defines the default channel used in the bundle.
+# Add a new line here if you would like to change its default config. (E.g DEFAULT_CHANNEL = "stable")
+# To re-generate a bundle for any other default channel without changing the default setup, you can:
+# - use the DEFAULT_CHANNEL as arg of the bundle target (e.g make bundle DEFAULT_CHANNEL=stable)
+# - use environment variables to overwrite this value (e.g export DEFAULT_CHANNEL="stable")
 DEFAULT_CHANNEL=1.x
```

Comment:
wouldn't be better to already start to have channels for the spedific version, 1.10.x, 1.11.x etc, it would make easier to do upgrades later.

Reply:
I had created below Jira to incorporate it.
https://issues.redhat.com/browse/KOGITO-5836

Reply:
thanks :)